contents/docs/surveys/branching-logic.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Branching logic
+sidebar: Docs
+showTitle: true
+availability:
+  free: none
+  selfServe: full
+  enterprise: full
+---
+
+It is useful to be able to display survey questions conditionally. For example, you may want to ask certain questions based on the responses received. Branching logic allows you to do this.
+
+## Simple branching
+
+For every question type, you can now define the next step to navigate to after a question is completed. Click on __After this question, go to:__, then choose the next step you want to transition to. The options are:
+
+- Next question
+- Any specific question
+- Confirmation message (if present) or End
+
+After the given question is completed, the chosen next step will be followed.
+
+## Response-based branching
+
+For the __Single choice__ and __Rating__ question types, you can define the next step based on the response received. Click on __After this question, go to:__, and choose the option __Specific question based on answer__. Depending on your question type, this will present the following options.
```

Comment:
```suggestion
For the **single choice** and **rating** question types, you can define the next step based on the response received. 

Click on **After this question, go to:**, and then choose **Specific question based on answer**. Depending on your question type, this will present the following options:
```